lib/fml_forms/validations.rb

```diff
@@ -10,6 +10,7 @@ module FML
   #   Given a spec like:
   #    form:
   #      title: "Validation Example"
+  #      id: 0acd2dc0b8b325eb5698e50387b79d00d77b1f10
```

Comment:
1. I cannot figure any way to do this without giving exams unique ids

2. I gave them SHA hashes, but the ids can be anything that doesn't change, so it should still be doable by non-technical person. The id for an exam could, for example, be identical to its title (except it shouldn't change when/if the title does. I should add documentation for this)

3. I can't generate them programatically, because the exams can be generated by a person with a text editor; there's no point at which FML has write access to the exam